### PR TITLE
Enable `./minikube logs -f`

### PR DESCRIPTION
**Pull request by lichen2013**
Fixes issue https://github.com/kubernetes/minikube/issues/2447

### CODE REVIEW

**Comment by dlorenc** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by lichen2013** [comment]
Are the checks failed due to my change ?
I can not get why the failure, anyone can help ?

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by balopat** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by tstromberg** [comment]
Oof. I just ran into the fact that this is broken today. 

Do you mind addressing the merge conflict? I'd like to see about merging this if it's still viable. My apologies for the unnecessarily long delay in reviewing this PR.

**Comment by lichen2013** [comment]
let me have a check.

File: pkg/minikube/bootstrapper/ssh_runner.go [hunk 1]
```diff
@@ -63,32 +64,46 @@ func (s *SSHRunner) Run(cmd string) error {
...
-	defer sess.Close()
-
-	b, err := sess.CombinedOutput(cmd)
+	var b singleWriter
+	err := s.CombinedOutputTo(cmd, &b)
 	if err != nil {
 		return "", errors.Wrapf(err, "running command: %s\n.", cmd)
 	}
-	return string(b), nil
+	return string(b.b.Bytes()), nil
```
Comment (RA489) [inline pkg/minikube/bootstrapper/ssh_runner.go]: can we use some variable instead for better readilbility?

**Comment by tstromberg** [comment]
Any hope for resurrecting this PR from the merge conflicts? I feel like it's doing the right thing and would love to see it in minikube. Thanks for all the hard work you've put into it so far.

**Comment by tstromberg** [comment]
Thank you for the contribution. Due to recent inactivity, I am marking this PR as closed until the issues can be resolved.

Please re-open once this PR has been updated. Thank you again!

**Pull request closed without merging**